bftengine/src/bftengine/messages/InternalMessage.hpp

```diff
@@ -11,18 +11,44 @@
 
 #pragma once
 
+#include <variant>
+#include "messages/FullCommitProofMsg.hpp"
+#include "messages/RetranProcResultInternalMsg.hpp"
+#include "SignatureInternalMsgs.hpp"
+
 namespace bftEngine::impl {
 
-class InternalReplicaApi;
+struct GetStatus {
```

Comment:
This seemed to have slipped in prematurely or was this copied from somewhere?

Reply:
Ah good point. Yes, this should only come in the follow up PR. i'll remove it.